### ISSUE TITLE
Add list of known private UIDs

### ISSUE THREAD

**Issue by anders9ustafsson** [fo-dicom/fo-dicom#589]
There are quite a large number of more or less official private UIDs for various types of storage. A comprehensive list of known UIDs can for example be found [here](http://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/specs/storage/storage.html).

For increased applicability of *fo-dicom*, it would be beneficial to add these private UIDs to the list of known UIDs.

Implementation-wise, it could be worthwhile considering to use T4 techniques to parse the UIDs from the referenced HTML document.

**Comment by BobSter3000** [comment]
Write Preview 
@anders9ustafsson I had a smaller list of Private UID's (DicomUidPrivate.cs) from and older abandoned fo-dicom project I was working on. I updated that file to include the UIDs from that web page. There was not that much of a difference, so I did not create a tt file. I did however put sort of a safety check in DicomUID.cs. I have the private uids assigned to `DicomStorageCategory.None.`
`if (Type != DicomUidType.SOPClass || !Name.Contains("Storage") || !UID.StartsWith("1.2.840.10008")) return DicomStorageCategory.None;`
I did this to prevent current implementations from accepting the private SOP classes by default, like in the samples SCP project. I personally think it better that a SCP implementations specifically accept a private SOP class.
If this is acceptable I can submit a pull request.
